src/shared.js

```diff
@@ -0,0 +1,10 @@
+export function verifyMethod(method) {
+  return ['POST', 'PUT', 'PATCH', 'DELETE'].includes(method);
```

Comment:
I think `Array.includes` is a risky API for browser compatibility. Perhaps storing the methods in an object literal and checking property existence might be preferable.